pkg/workflows/mapping.go

```diff
@@ -0,0 +1,92 @@
+package workflows
```

Comment:
Is this file extracted from models_yaml.go without changes? If so, do we need to do that? It just makes the PR larger for no reason. Plus it looks like you created a new "inputs" type inside models_yaml.go, instead of in a separate file? Maybe just keep mapping inside there too and refactor everything in a separate PR?

Reply:
Fair point!